server/lib/referenceLookup.js

```diff
@@ -26,6 +26,25 @@ function getEntity(req, res, next) {
   const reference = codeRef[2];
   const documentDefinition = identifiersIndex[code];
 
+  // hack: handle special case of stock movements
```

Comment:
Why does stock movements require a hack?

Reply:
the `referenceLookup.js` has a structure for handling reference like `CP.HBB.001` (Prefix.Project.Reference) but the stock a reference like `SM.1.1` (Prefix.FluxId.ComputedReference).

This file was not created for handling reference such as for stock movement, so we just put the code here for having the same result also for stock movement.

Reply:
Hmm..  then why does stock use `referenceLookup` if the reference is not similar?  Maybe it should use a different function?